package.json

```diff
@@ -137,6 +140,7 @@
     "eslint-plugin-storybook": "^0.11.3",
     "globals": "^16.0.0",
     "http-server": "^14.1.1",
+    "husky": "^9.1.7",
```

Comment:
Have you considered [lefthook](https://github.com/evilmartians/lefthook)? It runs faster and does not add any files to the repo

Reply:
umm,, interesting, i didn't know it before. checking it

Reply:
done. I'm fan of anything that is faster other solutions 😂

Reply:
BTW, if funny because it creates `.husky` folder